wondrous-app/components/organization/pods/constants.tsx

```diff
@@ -0,0 +1,5 @@
+export enum PodView {
+  ALL = 0,
```

Comment:
can we have a better name here? not sure what ALL stands for but I assume it's for all users? so we can do ALL_USERS or something like that ? something more descriptive - helps when searching